mlflow/onnx.py

```diff
@@ -0,0 +1,181 @@
+            }
+            predicted = self.rt.run(self.output_names, feed_dict)
+        else:
+            values = dataframe.values
+            if values.dtype == np.float64:
+                values = values.astype(np.float32)
+            predicted = self.rt.run(self.output_names, {self.input_names[0]: values})
+        return pd.DataFrame.from_dict(
+            {c: p.reshape(-1) for (c, p) in zip(self.output_names, predicted)})
+
+
+def _load_pyfunc(path):
+    """
+    Load PyFunc implementation. Called by ``pyfunc.load_pyfunc``.
+    """
+    return _OnnxModelWrapper(path)
+
+
+def load_model(model_uri):
+    """
+    Load an ONNX model from a local file (if ``run_id`` is None) or a run.
+
+    :param model_uri: The location, in URI format, of the MLflow model, for example:
+
+                      - ``/Users/me/path/to/local/model``
+                      - ``relative/path/to/local/model``
+                      - ``s3://my_bucket/path/to/model``
+                      - ``runs:/<mlflow_run_id>/run-relative/path/to/model``
+
+                      For more information about supported URI schemes, see the
+                      `Artifacts Documentation <https://www.mlflow.org/docs/latest/tracking.html#
+                      supported-artifact-stores>`_.
+
+    :return: An ONNX model instance.
+
+    """
+    local_model_path = _download_artifact_from_uri(artifact_uri=model_uri)
+    flavor_conf = _get_flavor_configuration(model_path=local_model_path, flavor_name=FLAVOR_NAME)
+    onnx_model_artifacts_path = os.path.join(local_model_path, flavor_conf["data"])
+    return _load_model(model_file=onnx_model_artifacts_path)
+
+
+def log_model(onnx_model, artifact_path, conda_env=None, **kwargs):
+    """
+    Log an ONNX model as an MLflow artifact for the current run.
+
+    :param onnx_model: ONNX model to be saved.
+    :param artifact_path: Run-relative artifact path.
+    :param conda_env: Either a dictionary representation of a Conda environment or the path to a
+                      Conda environment yaml file. If provided, this decribes the environment
+                      this model should be run in. At minimum, it should specify the dependencies
+                      contained in ``mlflow.onnx.DEFAULT_CONDA_ENV``. If `None`, the default
+                      ``mlflow.onnx.DEFAULT_CONDA_ENV`` environment will be added to the model.
+                      The following is an *example* dictionary representation of a Conda
+                      environment::
+
+                        {
+                            'name': 'mlflow-env',
+                            'channels': ['defaults'],
+                            'dependencies': [
+                                'python=3.6.0',
+                                'onnx=1.4.1',
+                                'onnxruntime=0.3.0'
+                            ]
+                        }
+
+    :param kwargs: kwargs to pass to ``onnx_model.save`` method.
```

Comment:
This `kwargs` parameter does not seem to be defined on `mlflow.onnx.save_model`. Additionally, the [onnx.save_model](https://github.com/onnx/onnx/blob/51b6ecce7a6f7bfc3b7944e77614bc0c38d0db96/onnx/__init__.py#L168) method doesn't take any keyword arguments that are currently used; here is the signature:
```
def save_model(proto, f, format=None):  # type: (Union[ModelProto, bytes], Union[IO[bytes], Text], Optional[Any]) -> None
    '''
    Saves the ModelProto to the specified path.
    @params
    proto should be a in-memory ModelProto
    f can be a file-like object (has "write" function) or a string containing a file name
    format is for future use
    '''
```

I think it makes sense to remove the `kwargs` parameter for now.